internal/engine/pod.go

```diff
@@ -34,49 +34,49 @@ func handlePodChangeAction(ctx context.Context, state *store.EngineState, action
 		return
 	}
 
-	pod := action.Pod
+	podInfo := action.Pod
```

Comment:
why `podInfo` and not just `pod`?

Reply:
It was an attempt to make the diff more readable (the old var that _actually_ got used was named `podInfo`), but it's more confusing than helpful so I just stuck with `pod`